src/server/lib/contributors.js

```diff
@@ -32,13 +32,18 @@ export async function fetchContributors({ collectiveSlug }) {
   // Fetch data from Open Collective API
   const query = gql`
     query GithubContributors($collectiveSlug: String) {
-      Collective(slug: $collectiveSlug) {
+      collective(slug: $collectiveSlug) {
```

Comment:
I'm not sure if this is relevant here but due to https://github.com/opencollective/opencollective/issues/6899 I couldn't test this towards the dev API. Instead I ran it against both `https://api.opencollective.com/graphql` and `https://api.opencollective.com/graphql/v2` and both of them defines this method as lowercase `collective`.

Similarly there's no field named `githubContributors` but a `githubContributors` witch contains contributor nodes.

I can revert these changes that's not related to `banner.js` if you can help me confirm it's not needed.

Reply:
Even though I never really got the dev API working I was browsing around the code. Seems like the [local V1 is using `Collective`](https://github.com/opencollective/opencollective-api/blob/bb2eae36c87cc400c7099509fd5149a85aa026fe/server/graphql/schemaV1.graphql#L4301-L4309) but neither ones posted above or V2 in the dev API.

Reply:
https://api.opencollective.com/graphql is defaulting to https://api.opencollective.com/graphql/v2, they're indeed the same thing. The legacy API is available as https://api.opencollective.com/graphql/v1 and the field is indeed exposed as `Collective` there.

Reply:
The `contributors` field is different: it's meant to surface your financial contributors on Open Collective, while `githubContributors` is the list of contributors to your repository. I suggest rolling back that part. I can then test and merge your changes.

Reply:
I reverted these changes! If it's not too much effort, do you mind a quick explanation on how you test this? As mentioned I had to create a real key and use the public API to test this but obviously I tested `v2` two times by mistake 🙈 

I opted to setup the test API with PostgreSQL 13 which is latest version I got working but it didn't work out of the box even though I was ensuring I was setting the right key that was set in the `.env`. Also is there another way other than querying the db for collectives to see what data is in the dump to test things like this? Something like a GraphQL Playground/Grapiql?